src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -508,7 +508,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
     getApproverWithQuery = debounce(
         (searchStr: string) =>
             this.getCollaborators(this.getApproverContactsSuccessCallback, this.errorCallback, searchStr, {
-                includeGroups: isFeatureEnabled(this.props.features, 'activityFeed.tasks.assignToGroup'),
+                includeGroups: true,
```

Comment:
does it make sense to preserve this as an exposed option, i.e., will there be cases where we want to get collaborators without groups?

Reply:
Hm from what I know I don't believe there will be a case where we want to dismiss groups functionality and only get collaborators